packages/atoms/src/Box.tsx

```diff
@@ -0,0 +1,62 @@
+import { D, pipe } from '@mobily/ts-belt';
+import { NativeStyleObject, prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Spacing } from '@trezor/theme';
+import React from 'react';
+import { View, ViewProps, ViewStyle } from 'react-native';
+
+const layoutStylePropsKeys = [
+    'flex',
+    'flexDirection',
+    'justifyContent',
+    'alignItems',
+    'alignContent',
+    'alignSelf',
+] as const;
+
+const spacingStylePropsKeys = [
+    'marginHorizontal',
```

Comment:
Looks to be way to flexible ;D That's exactly what @marekrjpolak was removing from modals

Reply:
Hmmm. Still sketchy. I think that props should be component-related not styles related, with some exception when some prop is used super often. If they are intended to be used extremely often then I get it. Let's see how it pans out :)